contents/blog/how-to-measure-product-engagement.md

```diff
@@ -0,0 +1,101 @@
+
+Churn is a metric we associate closely with customer satisfaction but alone it isn’t sophisticated enough to explain _why_ your customers do or do not stay. It’s also a lagging metric – it can take at least 30 days just to understand that a customer is not coming back, during which you’ve lost precious time.
+
+Your users are constantly providing feedback through their actions; which features they interact with, how they scroll, where their mouse hovers. By interpreting those actions, you’ll better understand what keeps them on your platform and be able to build more of what they need and love.
+
+Product engagement also makes a great [North Star Metric](https://posthog.com/blog/north-star-metrics), as it’s usually a reliable forecast for retention and can provide a strong foundation for product development.
+
+<BorderWrapper>
+    <Quote
+        imageSource="/images/customers/anca.png"
+        size="md"
+        name="Anca Filip"
+        title="Head of Product, Mention Me"
+        quote={`“PostHog has helped us improve our product and get a much better understanding of our users than we've ever been able to before.”`}
+    />
+</BorderWrapper>
+
+## What are product engagement metrics?
+The engagement metrics you need to track will depend on what your business looks like and what your product is.
+
+In order to use the right product engagement metric for your organization you need a strong understanding of your customer personas. What does your ideal user look like? How do they interact with your platform? What are the revenue drivers attached to them?
+By examining your personas and how their individual actions influence revenue and growth, you can select the most appropriate engagement metrics for your product.
+
+### Common engagement metrics
+For some products **interactions** are a key metric. Social media platforms, for example, look at numbers of likes, comments and shares.
+
+For a gaming product **session duration** is likely to be a good indicator of positive user engagement. The longer users play, the better. 
+
+Other digital services may track different types of usage. YouTube may base engagement metrics on **views**, for example, while Google may use the number of **search queries**.
+
+The individual metrics are limitless. You’ll see **pageviews**, **bounce rate**, **page scroll depth**, **purchases** and many more used as KPIs for different business models. It’s important not to choose these glibly, but to make sure you’ve chosen the perfect fit for your product.
+
+## What is the best way to measure product engagement?
+Listening to users’ actions is important, but how can you track engagement metrics and translate them into actionable insights?
+
+![](../images/blog/how-to-measure-engagement/correlation.png)
```

Comment:
I'd actually swap this and the next screenshot around - the next one is a nice overview that sets the scene visually